dask/array/core.py

```diff
@@ -1448,14 +1448,20 @@ def to_dask_dataframe(self, columns=None, index=None):
             Specifying ``index`` can be useful if you're conforming a Dask Array
             to an existing dask Series or DataFrame, and you would like the
             indices to match.
+        target_dataframe : DataFrame Instance, optional
+            An optional `target_dataframe` parameter can be passed for dask
+            to decide which type of DataFrame back-end to convert
```

Comment:
"convert to" makes it sound like we're creating an object of type `type(target_dataframe)`. I'd suggest something like

    to specify the concrete dataframe type to use for partitions of the Dask dataframe.
    By default, pandas DataFrame is used.

Reply:
@TomAugspurger Modified accordingly in rest of the methods.